mizar/common/rpc.py

```diff
@@ -21,13 +21,14 @@
 
 import logging
 import json
+import netifaces
 from mizar.common.common import run_cmd
 
 logger = logging.getLogger()
 
 
 class TrnRpc:
-    def __init__(self, ip, mac, itf='eth0', benchmark=False):
+    def __init__(self, ip, mac, itf=netifaces.interfaces()[1], benchmark=False):
```

Comment:
We shouldn't use second item in the netifaces, because it's not always the second interface is the VM's default network interface.